src/main/scala/introprog/PixelWindow.scala

```diff
@@ -254,6 +254,19 @@ class PixelWindow(
     Swing.await { new java.awt.Color(canvas.img.getRGB(x, y)) }
   }
 
+  import java.awt.image.BufferedImage
+  /**
+    * Returns a screenshot of the window
+    * SLOW! creates image one pixel at a time
```

Comment:
Don't shout in comments :).  I propose to remove the part that it is slow, as we do not offer any alternative anyway.